src/components/contact-govuk-one-login/contact-govuk-one-login-controller.ts

```diff
@@ -1,7 +1,24 @@
 import { Request, Response } from "express";
+import {
+  supportWebchatContact,
+  supportPhoneContact,
+  showContactGuidance,
+} from "../../config";
 
 export function contactGet(req: Request, res: Response): void {
-  const data = {};
+  const isAuthenticated = req.session?.user?.isAuthenticated;
```

Comment:
@danacotoran  have you tried this out?
As the RequiresAuthMiddleware is not called (in the root file) it's possible some people may be sent to this page from another system (where they are logged in) but we don't have the isAuthenticated field set in the session

Reply:
@DougsCER I tested it locally and it worked as expected (as in, I could see the "Sign out" link only when I was authenticated with my OL) but you're probably better positioned to identify the risks/issues to do with this approach. So if you would advise that this isn't a good idea, the UCD folks said this was a "good to have" so we could probably go without it.

Reply:
Berne thinking about this over the weekend. 
This approach will correctly show the sign-out link for those who have visited a page on AMF that requires authentication. 
Users that have signed in on another service then follow the link here won't be shown the sign-out link. This would be nice to do but not simple.  So I think we should progress as-is and consider if there is an improvement we can do in future to show the link for people who have signed in on another service.

Reply:
I agree - let's carry on as is and I'll make a tech debt story to remember this